node/src/remote/index.ts

```diff
@@ -236,7 +236,18 @@ export class RemoteTable<T = number[]> implements Table<T> {
   }
 
   async createIndex (indexParams: VectorIndexParams): Promise<any> {
-    throw new Error('Not implemented')
+    const data = {
+      column: indexParams.column,
+      index_type: 'vector',
+      metric_type: indexParams.metric_type
+    }
```

Comment:
Do we want to make `index_cache_size` configurable to match what we have on the python remote table?

Reply:
done!